service/apiserver/internal/logic/account/getaccountlogic.go

```diff
@@ -76,6 +77,9 @@ func (l *GetAccountLogic) GetAccount(req *types.ReqGetAccount) (resp *types.Acco
 		Nonce:  account.Nonce,
 		Assets: make([]*types.AccountAsset, 0, len(account.AssetInfo)),
 	}
+
+	totalAssetPrice := big.NewFloat(0)
```

Comment:
`price` seems to be not a good name. How about `totalAssetAmount` or `totalAssetValue`? Can you refer to some dapp (e.g., pancake, uniswap) for the name

Reply:
`totalAssetValue ` seems better

Reply:
Oh, OK.
Thanks 👍